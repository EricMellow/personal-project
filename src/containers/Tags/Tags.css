.tags {
background: url(../../assets/bricks1.jpg); 
height: 85vh;
display: flex;
flex-direction: column;
justify-content: center;
align-items: center;
background-repeat: no-repeat;
background-size: cover;
}

.tags-style {
  background-color: rgba(28, 141, 196, .9);
    display: flex;
    width: 675px;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 92%;
}

form .add-type-input {
    background: none;
    height: 3rem;
    border: 3px solid white;
    width: 580px;
    color: white;
    padding-left: 10px;
    margin: 0 0 20px 0;
    font-size: 2rem;
    border-radius: 5px;
}

.tags-form {
  width: 100%;
}