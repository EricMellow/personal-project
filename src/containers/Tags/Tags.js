import React, { Component } from "react";
import './Tags.css';
import MapContainer from "../MapContainer/MapContainer";

export class Tags extends Component {
  constructor(props) {
    super(props);
    this.state = {
      type: ''
    };
  }

  handleInput = (event) => {
    event.preventDefault();
    this.setState({
      [event.target.name]: event.target.value.toUpperCase()
    });
  }

  render() {

    return (
      <div className="tags ">
        <div className="tags-style" >
          <form className="tags-form">
            <input
              placeholder="Search for a type of activity"
              name="type"
              className="input add-type-input"
              value={this.state.type}
              onChange={event => this.handleInput(event)}
              type="text"
            />
          </form>
          <MapContainer type={this.state.type} />
        </div>
      </div>
    );
  }
}

export default Tags;