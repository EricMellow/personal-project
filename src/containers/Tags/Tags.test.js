import React, { Component } from "react";
import './Tags.css';
import { connect } from 'react-redux';

export class Tags extends Component {
  

  render() {

    return (
      <div className="tags">
      LOOK AT ALL THESE #Tags #HASHTAG #OCTOTHORPE #POUNDSIGN
      </div>
    );
  }
}

export default Tags;