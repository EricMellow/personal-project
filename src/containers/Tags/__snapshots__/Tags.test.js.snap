// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Tags />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <form>
          <input
                    className="input type-input"
                    name="type"
                    onChange={[Function]}
                    placeholder="Search for a type of activity"
                    type="text"
                    value=""
          />
</form>,
        <Wrapper
          type=""
/>,
      ],
      "className": "tags",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <input
            className="input type-input"
            name="type"
            onChange={[Function]}
            placeholder="Search for a type of activity"
            type="text"
            value=""
/>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "input type-input",
            "name": "type",
            "onChange": [Function],
            "placeholder": "Search for a type of activity",
            "type": "text",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        "type": "form",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "type": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <form>
            <input
                        className="input type-input"
                        name="type"
                        onChange={[Function]}
                        placeholder="Search for a type of activity"
                        type="text"
                        value=""
            />
</form>,
          <Wrapper
            type=""
/>,
        ],
        "className": "tags",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <input
              className="input type-input"
              name="type"
              onChange={[Function]}
              placeholder="Search for a type of activity"
              type="text"
              value=""
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "input type-input",
              "name": "type",
              "onChange": [Function],
              "placeholder": "Search for a type of activity",
              "type": "text",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          "type": "form",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "type": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
